src/parser.yy

```diff
@@ -316,6 +325,10 @@ map : MAP               { $$ = new ast::Map($1, @$); }
 var : VAR { $$ = new ast::Variable($1, @$); }
     ;
 
+variable : var { $$ = $1; }
```

Comment:
Maybe something like `var_or_map` instead? The name "variable" confused me when I first read through, thinking it didn't include maps.

Reply:
My issue with `map_or_var` is that once a 3rd type of variable gets added all those cases have to be renamed. But I guess that is unlikely to happen. I'll rename it